management-api-common/src/main/java/com/datastax/mgmtapi/shims/CassandraAPI.java

```diff
@@ -82,4 +82,8 @@ default Object handleRpcResult(Callable<Object> rpcResult) throws Exception {
   default List<String> getKeyspaces() {
     return StorageService.instance.getKeyspaces();
   }
+
+  default void reloadTrustManager() throws Exception {
+    throw new Exception("Unimplemented for Cassandra, only available for DSE");
```

Comment:
suggestion: UnsupportedOperationException instead?

Reply:
That is also RuntimeException (since this is a Runtime issue) so the method does not have to declare throw Exception.

Reply:
I agree with this.

Reply:
I'll see what I can do.